### ISSUE TITLE
Setting _FillValue attribute

### ISSUE THREAD

**Issue by jlumsden-mts** [predictionmachines/SDSlite#47]
First of all thanks for this library. We have been using it for some time to read NetCDF files but have just started writing them.

I'm trying to set the _FillValue attribute as per CF conventions (https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch02s05.html)

However, when setting this via the MetadataDictionary `variable.Metadata["_FillValue"] = 99999.0` it just results in an empty string ("") for _FillValue when examining the output using `ncdump -h` and causes errors (fails to open) when trying to open the file in a viewer like Panoply - I believe due to it not matching the datatype of the variable.

I think the issue stems from the fact that _FillValue is used internally by the NetCDF library and it wants it set prior to any data being put into the variable. 

Despite trying a few different permutations I could not get it to be set correctly using SDSLite. It would either not appear at all or be set to an empty string ("") rather than the double value I was expecting.

**Comment by vassilyl** [comment]
Did you try to set the `MissingValue` property on the variable instead?

**Comment by jlumsden-mts** [comment]
Yes, that works fine (setting "MissingValue" or "missing_value" attributes) but unfortunately would not be compliant with CF conventions - at least they consider it a deprecated attribute compared to _FillValue - see https://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch02s05.html

For context I am trying to match the convention used by these oceanographic glider data NetCDF files:
http://imos-data.s3-website-ap-southeast-2.amazonaws.com/?prefix=IMOS/ANFOG/slocum_glider/ScottReef320230424/
Sourced from https://portal.aodn.org.au/
